govc/vcenter/deploy.go

```diff
@@ -127,10 +155,17 @@ func (cmd *deploy) Run(ctx context.Context, f *flag.FlagSet) error {
 			DeploymentSpec: vcenter.DeploymentSpec{
 				Name:               vmName,
 				DefaultDatastoreID: datastoreID,
-				AcceptAllEULA:      true,
+				NetworkMapping: []vcenter.NetworkID{
+					vcenter.NetworkID{
+						Key:   "Network",
```

Comment:
Should `Key` value be `network.Reference().Type`, when network is a DVPG for example?